src/components/SortableTable/index.tsx

```diff
@@ -120,30 +121,20 @@ export const SortableTable = <DataItem extends Record<string, any>>({
             </tr>
           </thead>
           <tbody>
-            {localData[currentPage - 1].map((item, index) => {
-              const RowContents = () => (
-                <>
+            {localData[currentPage - 1].map(
+              (item, index) => (
+                /* eslint-disable react/no-array-index-key */
+                <tr
+                  className={item.href ? `SortableTable__clickableRow` : ""}
+                  key={`row-${index}`}
+                  onClick={() => (item.href ? history.push(item.href) : null)}
+                >
                   {hideNumberColumn ? null : <td>{index + 1}</td>}
                   {renderItemRow(item)}
-                </>
-              );
-
-              return (
-                // eslint-disable-next-line react/no-array-index-key
-                <tr key={`row-${index}`}>
-                  {item.href ? (
-                    <Link
-                      className="SortableTable__clickableRow"
-                      to={item.href}
-                    >
-                      <RowContents />
-                    </Link>
-                  ) : (
-                    <RowContents />
-                  )}
                 </tr>
-              );
-            })}
```

Comment:
I wanted use anchor tags for screen readers, but it was throwing valid DOMnesting error. Bummer